Add gatsby-plugin-copy to list of community plugins

ccummings opened the pull request:
Adding community plugin `gatsby-plugin-copy`

KyleAMathews commented:
Sweet thanks for sharing!

KyleAMathews commented:
Btw, curious why you went with this vs the static folder?

ccummings commented:
I'm running on Netlify and need different _headers and robots.txt files depending on the branch I'm deploying to. For everything else I use the static folder.

KyleAMathews commented:
Ah interesting! You might want to mention this use case in the README to differentiate it from the static folder.

ccummings commented:
Thanks for the feedback. I've added a note about why a user would use this plugin instead of the `static` folder. I also added an example that uses environment variables.

jlengstorf commented:
Hiya @ccummings! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs
